README/ReleaseNotes/v634/index.md

```diff
@@ -223,7 +225,18 @@ TCanvas::SaveAll({c1, c2, c3, c4}, "file.pdf");
 ```
 
 ## 2D Graphics Libraries
+* In `TGraphErrors` `TGraphAsymmErrors` and `TGraphBentErrors`, the error bars were drawn
+  inside the marker when the marker was bigger than the error bars. This produced a weird
+  plot. This is now fixed.
+
+ When error-bars exceeded the y range limits the end of error bars were nevertheless displayed
+  on the x-bottom and top axis. So it looked like the total error bar while it was indeed not.
+
+* New color schemes have been implemented as described [here](https://root.cern.ch/blog/new-color-schemes/).
```

Comment:
Maybe we could directly use the word "incusive" and cite the presentation at ICHEP?
What about texgyre fonts?

Reply:
Tex Gyre was already mentionned in v632  release notes